samples/Basic_Serial/app/application.cpp

```diff
@@ -82,20 +82,20 @@ void init()
 	 * Functions debugf, debug_d, debug_i, debug_w, and debug_e store the format string
 	 * in flash so that the RAM is freed for more important information.
 	 *
-	 * Another useful feature is printing the filename and line number of every degub line
+	 * Another useful feature is printing the filename and line number of every debug line
 	 * This will require extra space on flash and can be enabled
 	 * using make parameter PRINT_FILENAME_AND_LINE=1*/
 
-	debug_d("(DEBUG) message printed on UART0");
+	debug_d("(DEBUG) message printed on UART0\n");
```

Comment:
In most debugf statements \n is appended to the format string. By applying it also by default in debug_progmem.h we get to the situation of double \n which makes reading the console harder.

So \\n was removed from the definition of debug_e and the user is responsible to put \n at the end if he wants (flexibility)
Alternative is to modify all debugf in the project to remove \n from the end of the format string